Providers/PythonProvider.cpp

```diff
@@ -548,13 +524,14 @@ PythonProvider::forkExec ()
     if (INVALID_SOCKET == m_FD)
     {
         int sockets[2];
+	int pid;
         int result = socketpair (AF_UNIX, SOCK_STREAM, 0, sockets);
         if (-1 != result)
         {
             // socketpair succeeded
             SCX_BOOKEND_PRINT ("socketpair - succeeded");
-            m_pid = fork ();
-            if (0 == m_pid)
+            pid = fork ();
+            if (0 == pid)
```

Comment:
it may be good to log the pid of the processes that we start, as well as when they complete

Reply:
yes but we don't log anything from this file. from python adaptor layer there is some more work we need to do to start logging from providers.